### PR TITLE
[Merged by Bors] - chore(AffineIsometry): weaken some Normed* to Seminormed*

### PR DESCRIPTION
**Pull request by urkud**
- Reorder `variable`s so that related variables
  are declared on the same line;
- Use `Seminormed*`/`PseudoMetric*` for all non-primed type variables.
- Introduce `V₁'`/`P₁'` for a normed add torsor.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Analysis/NormedSpace/AffineIsometry.lean [hunk 1]
```diff
@@ -39,11 +39,12 @@ algebra-homomorphisms.)
...
-  [SeminormedAddCommGroup V₃] [SeminormedAddCommGroup V₄] [NormedSpace 𝕜 V] [NormedSpace 𝕜 V₁]
-  [NormedSpace 𝕜 V₂] [NormedSpace 𝕜 V₃] [NormedSpace 𝕜 V₄] [PseudoMetricSpace P] [MetricSpace P₁]
-  [PseudoMetricSpace P₂] [PseudoMetricSpace P₃] [PseudoMetricSpace P₄] [NormedAddTorsor V P]
-  [NormedAddTorsor V₁ P₁] [NormedAddTorsor V₂ P₂] [NormedAddTorsor V₃ P₃] [NormedAddTorsor V₄ P₄]
+  [NormedField 𝕜]
+  [SeminormedAddCommGroup V] [NormedSpace 𝕜 V] [PseudoMetricSpace P] [NormedAddTorsor V P]
+  [SeminormedAddCommGroup V₁] [NormedSpace 𝕜 V₁] [MetricSpace P₁] [NormedAddTorsor V₁ P₁]
+  [SeminormedAddCommGroup V₂] [NormedSpace 𝕜 V₂] [PseudoMetricSpace P₂] [NormedAddTorsor V₂ P₂]
+  [SeminormedAddCommGroup V₃] [NormedSpace 𝕜 V₃] [PseudoMetricSpace P₃] [NormedAddTorsor V₃ P₃]
+  [SeminormedAddCommGroup V₄] [NormedSpace 𝕜 V₄] [PseudoMetricSpace P₄] [NormedAddTorsor V₄ P₄]
```
Comment (eric-wieser) [inline Mathlib/Analysis/NormedSpace/AffineIsometry.lean:47]: Note in Lean 3 we wrote this as:
```lean
  [normed_field 𝕜] [seminormed_add_comm_group V] [seminormed_add_comm_group V₁]
  [seminormed_add_comm_group V₂] [seminormed_add_comm_group V₃]
    [seminormed_add_comm_group V₄]
  [normed_space 𝕜 V] [normed_space 𝕜 V₁] [normed_space 𝕜 V₂] [normed_space 𝕜 V₃]
    [normed_space 𝕜 V₄]
  [pseudo_metric_space P] [metric_space P₁] [pseudo_metric_space P₂] [pseudo_metric_space P₃]
    [pseudo_metric_space P₄]
  [normed_add_torsor V P] [normed_add_torsor V₁ P₁] [normed_add_torsor V₂ P₂]
    [normed_add_torsor V₃ P₃] [normed_add_torsor V₄ P₄]
```
ie, keeping similar assumptions grouped. Unfortunately the line breaks were dropped by mathport

File: Mathlib/Analysis/NormedSpace/AffineIsometry.lean [hunk 2]
```diff
@@ -39,11 +39,12 @@ algebra-homomorphisms.)
...
 
 variable (𝕜 : Type*) {V V₁ V₂ V₃ V₄ : Type*} {P₁ : Type*} (P P₂ : Type*) {P₃ P₄ : Type*}
-  [NormedField 𝕜] [SeminormedAddCommGroup V] [SeminormedAddCommGroup V₁] [SeminormedAddCommGroup V₂]
-  [SeminormedAddCommGroup V₃] [SeminormedAddCommGroup V₄] [NormedSpace 𝕜 V] [NormedSpace 𝕜 V₁]
-  [NormedSpace 𝕜 V₂] [NormedSpace 𝕜 V₃] [NormedSpace 𝕜 V₄] [PseudoMetricSpace P] [MetricSpace P₁]
-  [PseudoMetricSpace P₂] [PseudoMetricSpace P₃] [PseudoMetricSpace P₄] [NormedAddTorsor V P]
-  [NormedAddTorsor V₁ P₁] [NormedAddTorsor V₂ P₂] [NormedAddTorsor V₃ P₃] [NormedAddTorsor V₄ P₄]
+  [NormedField 𝕜]
+  [SeminormedAddCommGroup V] [NormedSpace 𝕜 V] [PseudoMetricSpace P] [NormedAddTorsor V P]
+  [SeminormedAddCommGroup V₁] [NormedSpace 𝕜 V₁] [MetricSpace P₁] [NormedAddTorsor V₁ P₁]
```
Comment (eric-wieser) [inline Mathlib/Analysis/NormedSpace/AffineIsometry.lean:44]: Can we leave this as `PseudoMetricSpace` and introduce a `P₁'` for the MetricSpace? Some of the docstrings still refer to `P₁`, and I think it's more symmetrical to use `P₁` and `P₂` than `P` and `P₂`.

File: Mathlib/Analysis/NormedSpace/AffineIsometry.lean (hunk 1 above)
Comment (urkud) [reply Mathlib/Analysis/NormedSpace/AffineIsometry.lean:47]: I intentionally changed from "similar typeclasses go together" to "each object is declared on a dedicated line". Do you want me to revert it?

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**